float64 x_ref_w
float64 y_ref_w
---
bool  update_status